shard.lock

```diff
@@ -2,9 +2,9 @@ version: 2.0
 shards:
   amq-protocol:
     git: https://github.com/cloudamqp/amq-protocol.cr.git
-    version: 1.1.14
+    version: 1.1.15
```

Comment:
I would like to see the dependency change in a separate commit, unless they are needed for this PR of course.